torch/_decomp/decompositions.py

```diff
@@ -21,6 +21,7 @@
 )
 from torch.fx.experimental.symbolic_shapes import guard_int
 from torch.utils._pytree import tree_flatten, tree_map
+from torch.ao.quantization.fx._decomposed import quantized_decomposed_lib
```

Comment:
Refactor. 
Decomps for custom op doesn't need to live in this file. User should be able to write decomp in their user script.